modules/base/spec/classes/collectd_spec.rb

```diff
@@ -72,7 +72,7 @@
     is_expected_to_have_plugins('/etc/collectd.d/graphite.conf', %w(write_graphite))
     is_expected.to  contain_file('/etc/collectd.d/graphite.conf')
         .with_content(/\s+Host "some-host"$/)
-        .with_content(/\s+Port "some-port"$/)
+        .with_content(/\s+Port "1337"$/)
```

Comment:
Out of curiosity, why 1337? Is it a default port for graphite?

Reply:
It was a tongue-in-cheek "randomly" chosen number to use as test data, a reference to leet-speak :)

Reply:
Interesting! I'd assumed it was a reference the start of the Hundred Years War.